pcicrawler/cli.py

```diff
@@ -316,6 +324,10 @@ def main(
         no_scripting()
         # When asked to print a tree, include filtered devices parents
         print_tree(sorted(devs, key=lambda d: d.device_name))
+    elif ocp:
+        run = tv.TestRun(name="pcicrawler", version=pkg_resources.get_distribution("pcicrawler").version)
```

Comment:
i dont quite understand why you need `pkg_resources` here. you can just use the `VERSION` or `__version__` from the package

Reply:
I believe this is the official pythonic way of doing it? either way should work fine